problem_builder/mentoring.py

```diff
@@ -979,7 +977,6 @@ def student_view(self, context):
             'children_contents': children_contents,
         }))
         fragment.add_css_url(self.runtime.local_resource_url(self, 'public/css/problem-builder.css'))
-        fragment.add_css_url(self.runtime.local_resource_url(self, 'public/css/lms.css'))
```

Comment:
If we're removing this `lms.css` CSS file, then why bother keeping it (and editing it) below?

Reply:
Oh sorry! I cherry-picked the v2 changes here, but I guess that doesn't delete files. I'll delete lms.css